src/re_frame_tracer/core.cljs

```diff
@@ -42,34 +42,35 @@
           (let [title (if protocol
                         (str protocol " " name " " arglist)
                         (str ns "/" name
+                             (when tag (str " " tag))
                              (when dispatch-val
                                (str " " (pr-str dispatch-val)))
                              (str " " arglist)
                              (when anonymous? " (anonymous)")))
                 arglist (remove '#{&} arglist)]
             (.groupCollapsed js/console "%c%s" (str "color:" color ";") title)
-            (.groupCollapsed js/console "bindings"))
-          
+            (.group js/console "bindings"))
```

Comment:
I suspect we will want to keep this as `groupCollapsed`, will defer to @stumitchell though.